x-pack/legacy/plugins/graph/public/app.js

```diff
@@ -173,7 +173,22 @@ app.controller('graphuiPlugin', function (
 
   function handleError(err) {
     return checkLicense(Promise, kbnBaseUrl)
-      .then(() => notify.error(err));
+      .then(() => {
+        const toastTitle = i18n.translate('xpack.graph.errorToastTitle', {
+          defaultMessage: 'Graph Error',
+          description: '"Graph" is a product name and should not be translated.',
+        });
+        if (err instanceof Error) {
+          toastNotifications.addError(err, {
+            title: toastTitle,
+          });
+        } else {
+          toastNotifications.addDanger({
```

Comment:
When `err` is an object and no instance of Error, I get the following Exception

![Bildschirmfoto 2019-07-11 um 09 58 37](https://user-images.githubusercontent.com/463851/61033733-58c90f80-a3c4-11e9-9c92-4271577ca3c9.png)

The message that should be displayed in this case is `err.data.message`

![Bildschirmfoto 2019-07-11 um 10 06 54](https://user-images.githubusercontent.com/463851/61033824-831acd00-a3c4-11e9-9b12-0c3825189a7f.png)

Reply:
you can reproduce it by selecting a ES field of type `text`

Reply:
Before:
![Bildschirmfoto 2019-07-11 um 09 52 21](https://user-images.githubusercontent.com/463851/61048597-8e7cf100-a3e2-11e9-8d18-7f42fcf50cba.png)
After:
![Bildschirmfoto 2019-07-11 um 13 46 13](https://user-images.githubusercontent.com/463851/61048602-93da3b80-a3e2-11e9-8e94-151fa8a13f51.png)